K2Bridge.Tests.UnitTests/KustoDAL/HitsMapperTests.cs

```diff
@@ -10,6 +10,7 @@ namespace UnitTests.K2Bridge.KustoDAL
     using System.Data.SqlTypes;
     using System.Globalization;
     using System.IO;
+    using System.Linq;
     using FluentAssertions;
```

Comment:
Can you please remove this as it is unused?

Reply:
I believe `using FluentAssertions` and `using FluentAssertions.Json` overlap, don't they?

Reply:
Ah. sorry. Yes, sure.